api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessageType.java

```diff
@@ -1,7 +1,7 @@
 package uk.gov.dhsc.htbhf.claimant.message;
 
 public enum MessageType {
-    CREATE_NEW_CARD,
+    REQUEST_NEW_CARD,
```

Comment:
Strictly speaking we should have some SQL to migrate CREATE_NEW_CARD messages to REQUEST_NEW_CARD. However we don't have any live claims so we can probably get away without it.